docker/python2/Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM nvidia/cuda:7.5-cudnn5-devel
+FROM nvidia/cuda:8.0-cudnn6-devel
 
 RUN apt-get update -y && \
     apt-get install -y --no-install-recommends \
     python-dev \
     python-pip && \
     rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*
 
+RUN pip install -U pip wheel
```

Comment:
Is this line needed?

Reply:
It is likely `cupy` and `chainer` required to install `wheel` because I have got this kind of error:
```
error: invalid command 'bdist_wheel'
```

It also give a warning to upgrade pip version. (https://pip.pypa.io/en/stable/news/)
But if you want to keep the original pip from Ubuntu distribution then I can remove pip upgrade and it would be only:

```
RUN RUN pip3 install wheel cupy==4.0.0b1 chainer==4.0.0b1
```